src/pages/sustainability/Sustainability.module.css

```diff
@@ -0,0 +1,18 @@
+.blurb {
+  @mixin body-3;
+  width: 100%;
+  margin-bottom: 80px;
+
+  @media (--tablet) {
+    @mixin body-1;
+  }
+
+  p:not(:last-of-type) {
+    margin-bottom: 1.18em;
```

Comment:
Using such small decimal relative units like this feels confusing. In general the new layout has preferred px over em in most situations – though I can see a case for still using ems if it was a reusable mixin/component that could be used in different font size contexts.

Reply:
The font-size of the `p` tag changes based on breakpoints. Going with an `em` in this case, makes the margin adjust automatically when the `p` fontsize changes.

Defining this in pixels would mean that I would have to adjust the margin also when the breakpoint changes. I can round it up to `1.2` tho, im sure the designers wouldn't mind.